gateway/assets/script/bootstrap.js

```diff
@@ -143,7 +143,7 @@ app.controller("home", ['$scope', '$log', '$http', '$location', '$timeout', '$md
             var previous = $scope.functions;
 
             var cl = function(previousItems) {
-                $http.get("/system/functions").then(function(response) {
+                $http.get("../system/functions").then(function(response) {
```

Comment:
is there any likelihood that nginx, caddy or some other ingress could block a relative path like "../"? cc @stefanprodan

Reply:
It should be fine, if it works with kube proxy it should also work with nginx sub paths

Reply:
Thanks for your advice. I'll merge.